spendsterIOS/Presentation/HomeScreen/Profile/ProfileViewController.swift

```diff
@@ -0,0 +1,56 @@
+//
+//  ProfileViewController.swift
+//  spendsterIOS
+//
+//  Created by Dmytro Holovko on 5/2/19.
+//  Copyright © 2019 KeyToTech. All rights reserved.
+//
+
+import UIKit
+
+class ProfileViewController: UIViewController {
+    @IBOutlet weak var scroller: UIScrollView!
+    @IBOutlet weak var profileImage: CornerImage!
+    @IBOutlet weak var username: UILabel!
+    @IBOutlet weak var email: UILabel!
+    let defaults = UserDefaults.standard
```

Comment:
@Thongvor userDefaults, as a store should be used from presenter, not here

Reply:
done